seqr/views/apis/saved_variant_api_tests.py

```diff
@@ -704,7 +704,7 @@ def test_update_saved_variant_json(self, mock_get_variants):
         url = reverse(update_saved_variant_json, args=['R0001_1kg'])
         self.check_manager_login(url)
 
-        response = self.client.get(url)
+        response = self.client.post(url)
```

Comment:
Is there any reason for changing `get` to `post`?

Reply:
yeah I changed the client side code to do a POOST so it is CSRF verified so I changed the test to reflect that